requirements.txt

```diff
@@ -34,7 +34,7 @@ cssselect==0.9.1
 splinter==0.7.5
 tblib==1.3.0
 selenium==2.53.6
-django-filer==1.2.5.post1
```

Comment:
OK. Puoi tirare dentro l'ultima versione di django-filer, in modo che sia possibile fare una PR? Sarei felice anche solo se la nostra versione sia unificabile alla attuale con una singola patch, poiche' ci permetterebbe di riapplicarla piu' o meno facilmente nel caso di aggiornamenti importanti a django-filer.